src/otio_aaf_adapter/adapters/advanced_authoring_format.py

```diff
@@ -1638,27 +1663,15 @@ def read_from_file(
     return result
 
 
-def write_to_file(
-    input_otio,
-    filepath,
-    prefer_file_mob_id=False,
-    use_empty_mob_ids=False,
-    **kwargs
-):
+def write_to_file(input_otio, filepath, **kwargs):
```

Comment:
This undoes the work recently merged in #56.

Reply:
Oops. Fixed.